spec/features/home_spec.rb

```diff
@@ -5,7 +5,7 @@
    	visit root_path
 
 		click_link 'Blog'
-		
-		expect(page).to have_content 'The Home Page'
+
+		expect(page).to have_content 'My Cool Blog Name Here'
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.